src/components/Layout/Layout.js

```diff
@@ -1,12 +1,23 @@
 import React from "react";
+import { useSelector } from "react-redux";
 import PrimaryNav from "components/PrimaryNav/PrimaryNav";
+import classNames from "classnames";
+import useHover from "hooks/useHover";
+
+import { isSidebarCollapsible } from "app/selectors";
 
 import "./_layout.scss";
 
 const Layout = ({ children }) => {
+  const [sidebarRef, isSidebarHovered] = useHover();
+  const sidebarCollapsible = useSelector(isSidebarCollapsible);
```

Comment:
For the implemented use case I'd say that this should be done using a prop passed from the parent page component being rendered instead of storing this in the redux state. This is because the sidebar on `Models` doesn't care what the sidebar on `Details` did. 

However if we want to implement some sidebar pinning that would persist through pages this implementation makes it quite trivial to do so. It also gives us a great way for different parts of the application to have knowledge available about the UI state of the application.